File: 백준/Bronze/7567. 그릇/그릇.js
const fs = require('fs');
// let input = fs.readFileSync('./input.txt').toString().trim().split('\n');

/**정답 제출 시에는 이 코드로 제출 */
let input = fs.readFileSync('/dev/stdin').toString().trim().split('\n');

const arr = input[0].split('');
let answer = 0;

for (let i=0; i<arr.length; i++) {
    if (i === 0) {
        answer += 10;
    }
    else if (arr[i] === arr[i-1]) {
        answer += 5;
    } else {
        answer += 10;
    }
}

console.log(answer);

File: 백준/Bronze/7567. 그릇/그릇.py
import sys
input = sys.stdin.readline

arr = input().rstrip()
answer = 0

for i in range(len(arr)):
    if i == 0:
        answer += 10
    elif arr[i-1] == arr[i]:
        answer += 5
    else:
        answer += 10

print(answer)